sw/Modules/Comms/Inc/can_interface.h

```diff
@@ -17,17 +17,13 @@
 extern "C" {
 #endif
 
-/* start the can bus and transmit/receive routine */
-void can_start_basic(void);
-
-/* the main can loop that handles transmission and initiate receive callback */
-void can_main(void);
+extern CanardInstance canard;
 
 /* this is the functions that goes before the while(1) */
-void can_main_setup(void);
+void setup_comms(void);
 
 /* this is the function that goes inside the while(1) */
-void can_main_loop(void);
+void loop_comms(void);
```

Comment:
you could name this `run_comms_rate_unlimited` or `run_comms_100hz` or `run_comms_task_100hz`.

This is fine too.